theme/views.py

```diff
@@ -4,6 +4,7 @@
 from django.views.generic import TemplateView
 from django.http import HttpResponse
 from django.shortcuts import redirect
+from django.conf import settings
```

Comment:
There is the import statement "from mezzanine.conf import settings" below this added "from django.conf import settings" and I believe this added settings import from django.conf will create conflict with mezzanine import. It does not appear to me specific settings from django.conf is used in this file. If I missed something and it is indeed needed, I believe an alias for settings has to be created so it will not conflict with imported settings from Mezzanine.